### ISSUE TITLE
Alternative to derivedattribute?

### ISSUE THREAD

**Issue by dmclark** [nicolaskruchten/pivottable#957]
In am trying to have a derivedattribute like:
`var makeLink = function(record) {
	var url = window.location.href.replace('admin/report','test/') + record.test_id;
	return '<a href="' + url +'" target="_blank">' + record.test_id + '</a>';
	};`

It'll only work if i undo [https://github.com/nicolaskruchten/pivottable/pull/401](401)

is there a more appropriate way to do this?

**Comment by nicolaskruchten** [comment]
No, that would be the way to do it. I'm not supporting HTML rendering in cells any more, so if you need it, you'll need to fork the table renderer.
